InfinityCosmos/ForMathlib/AlgebraicTopology/SimplicialSet/MorphismProperty.lean

```diff
@@ -26,4 +27,35 @@ def trivialFibration : MorphismProperty SSet := fun _ _ p ↦ BoundaryInclusions
 
 end trivialFibration
 
+section isoFibration
+
+/-- Inductive definition of inner horn inclusions Λ[n, i] ⟶ Δ[n] 
+  by restricting general horn inclusions to 0 < i < n -/
+inductive InnerInclusion : {X Y : SSet} → (X ⟶ Y) → Prop
+  | mk (n i : ℕ) (low : 0 < i) (high : i < n) : InnerInclusion (hornInclusion n i)
+
+/-- The class of inner horn inclusions as a MorphismProperty -/
+def InnerInclusions : MorphismProperty SSet := fun _ _ p ↦ InnerInclusion p
```

Comment:
For readability later, perhaps these should be called "InnerHornInclusions"? (I'm unclear about capitalization; see discussion on zulip.)

Reply:
Okay, I followed the discussion on Zulip, and I am still unsure what the capitalization should be, so I'll keep that as is for now and just adjust the name as suggested.